crates/uniswapx-rs/src/order.rs

```diff
@@ -8,6 +8,13 @@ use anyhow::Result;
 
 use crate::sol_math::MulDiv;
 
+fn current_timestamp() -> u64 {
+    std::time::SystemTime::now()
+        .duration_since(std::time::UNIX_EPOCH)
+        .unwrap()
```

Comment:
nit: .expect() instead for clearer error message, or return a Result to gracefully handle